src/masternodes/mn_checks.cpp

```diff
@@ -2533,8 +2533,13 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
                         if (col.nTokenId == DCT_ID{0})
                             totalDFI += col.nValue;
 
-                    if (totalDFI < collateralsLoans.val->totalCollaterals / 2)
-                        return Res::Err("At least 50%% of the vault must be in DFI");
+                    if (static_cast<int>(height) < consensus.FortCanningHillHeight) {
+                        if (totalDFI < collateralsLoans.val->totalCollaterals / 2)
+                            return Res::Err("At least 50%% of the vault must be in DFI");
+                    } else {
+                        if (MultiplyAmounts(totalDFI, 100) < MultiplyAmounts(collateralsLoans.val->totalLoans, scheme->ratio) / 2)
+                            return Res::Err("At least 50%% of the vault must be in DFI");
```

Comment:
```suggestion
                            return Res::Err("At least 50% of the collateral must be in DFI");
```

Reply:
Thanks, I will fix it. we just have to keep the double % otherwise it works as a conversion specifier